client/sections.js

```diff
@@ -0,0 +1,7 @@
+/**
+ * Internal dependencies
+ */
+const config = require( 'config' ),
+	sections = require( config( 'project' ) );
```

Comment:
We expose array of sections as before. There is WebPack plugin which does some magic, but I don't know the details. @blowery, @mtias should help here.